public/display.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/skeleton/2.0.4/skeleton.css"
+    />
+    <title>Document</title>
+  </head>
+  <body>
+    <h1>DISPLAY</h1>
+    <!-- <p>Hello <%= user.charAt(0).toUpperCase() + user.slice(1)%></p> -->
+    <div class="container">
+      <div id="data">
+        <form action="/addWeblink", method="GET" >
+          <input type="text" name="name" id="name" placeholder="Enter Link Name" />
+          <input type="text" name="url" id="link" placeholder="Enter Link" />
+          <button type="submit" value="Submit">Add</button>
+        </form>
+      </div>
+      <form  method="GET" >
+        <table id="tableLinks">
+         <thead>
+          <tr>
+            <th></th>
+           <th>Name</th>
+           <th>Link</th>
+         </tr>
+         </thead>
+         <tbody id="tableLinksBody" >
+          <!-- <tr>
+            <td><input type="checkbox" name="id" value = "1" /></td>
```

Comment:
Remove commented code

Reply:
Hey, you put that... pls remove that on your own time!! Thanks